src/main/java/com/alipay/oceanbase/hbase/OHTable.java

```diff
@@ -209,6 +230,38 @@ public OHTable(Configuration configuration, String tableName) throws IOException
         finishSetUp();
     }
 
+    // set Hbase table and OBKV-Hbase table mapping relationship
+    private void setNamespaceMap(Configuration conf) {
```

Comment:
maybe we can construct param_url in ObTableClient getOrCreateObTableClient instead of relying on configuration only?

Reply:
I think this will be better, but it requires parameters such as ip and port

Reply:
I think this will be better, but it requires parameters such as ip and port that are not in conf